README.md

```diff
@@ -1 +1 @@
-# We-connect
```

Comment:
Great job in putting the description of your app. 
Try to put more details like how to install, start and test your app.